Shift3/boilerplate-client-angular
# feat, refactor, test(app): translate client error messages

## Changes

- [x] Add the static error messages to the language files.
- [x] Update `SentryConfig` to accept a message of `Message` instead of `string`.
- [x] Pass the language keys instead of hard coded strings.
- [x] Update the associated tests.

## Purpose

The webapp should translate all error messages from the client, including hard coded messages (for things like unhandled exceptions from the server or no internet connection).

#### If you are not a member of this project, _skip this step_

_How do the users test this change?_

  1. Load the site.
  2. Trigger error notifications (there is currently a 500 when trying to recreate an agency that has been soft-deleted).
  3. Confirm that the errors show correctly in the toasts.
  4. Cycle through other languages.
  5. Confirm the messages appear in the correct language.

Closes #277.
